src/commons/listener.rs

```diff
@@ -67,10 +70,12 @@ pub enum ServiceType {
 
 /// Exposes a set of pods to the outside world.
 ///
-/// Essentially a Stackable extension of a Kubernetes [`Service`]. Compared to [`Service`], [`Listener`] changes three things:
-/// 1. It uses a cluster-level policy object ([`ListenerClass`]) to define how exactly the exposure works
+/// Essentially a Stackable extension of a Kubernetes Service. Compared to Service, Listener changes three things:
+/// 1. It uses a cluster-level policy object (ListenerClass) to define how exactly the exposure works
 /// 2. It has a consistent API for reading back the exposed address(es) of the service
-/// 3. The [`Pod`] must mount a [`Volume`] referring to the `Listener`, which also allows us to control stickiness
+/// 3. The Pod must mount a Volume referring to the Listener, which also allows us to control stickiness.
```

Comment:
You mean this https://docs.stackable.tech/home/stable/listener-operator/listener#_sticky_scheduling? We could add the link here.
```suggestion
/// 3. The Pod must mount a Volume referring to the Listener, which also allows "sticky" scheduling.
```

Reply:
good idea, done!